visuals/js/index.js

```
@@ -159,56 +172,30 @@ function gameLoop() {
-        // find element with data-value = item.type
-        const itemElem = teamInfoContainerElems[index].querySelector(
-          `[data-value="${item.type}"]`
-        );
-
-        // if itemElem is not present, skip
-        if (!itemElem) return;
-
-        itemElem.style.filter = "none";
-      });
+      teamInfoContainerElems[index].querySelector(".team_cells").textContent =
+        "Cells: " + player.cells.length;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)